pkgs/tools/archivers/fsarchiver/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig
+, zlib, bzip2, lzma, lzo, lz4, zstd, xz
+, libgcrypt, e2fsprogs, utillinux, libgpgerror }:
+
+let
+  version = "0.8.5";
+
+in stdenv.mkDerivation rec {
```

Comment:
why a `rec` here if `version` is in a `let` block?

Reply:
Because I idn't know that it wasn't needed if you had a let-block. Removed now.